docs/reference/pip_install.rst

```diff
@@ -82,20 +82,12 @@ Requirements File Format
 Each line of the requirements file indicates something to be installed,
 and like arguments to :ref:`pip install`, the following forms are supported::
 
-    <requirement specifier>
     <requirement specifier> [--install-option="..."] [--global-option="..."]
```

Comment:
my thinking was that the specifier *includes* the possibility of a marker, so it shouldn't be additional syntax to document here, but rather to be mentioned in the section on requirement specifiers.

but on 2nd thought,  I don't think that's conceptually accurate, so I  can reshuffle again.